event/aggregator_test.go

```diff
@@ -60,7 +55,18 @@ func aggregateSlowLog(input, output string, utcOffset time.Duration, examples bo
 		a.AddEvent(e, id, f)
 	}
 	got := a.Finalize()
-	gotJson, err := json.Marshal(got)
+	gotJson, err := json.MarshalIndent(got, "", "  ")
+	if err != nil {
+		l.Fatal(err)
```

Comment:
I wonder if we should stop mixing `testing` and testify in the same file at least for new code?

Reply:
Not sure what you mean here, `l.Fatal(err)` refers here to `log` package, beside it's an old code, I've added indent only.

Reply:
Ah, that helper does not accept `testing.T`. Well, ok.